src/Admin/BlockAdmin.php

```diff
@@ -154,7 +154,7 @@ protected function configureFormFields(FormMapper $form): void
             $containerBlockTypes = $this->containerBlockTypes;
 
             // need to investigate on this case where $page == null ... this should not be possible
-            if ($isStandardBlock && null !== $page && [] === $containerBlockTypes) {
+            if ($isStandardBlock && null !== $page && [] !== $containerBlockTypes) {
```

Comment:
I made a mistake when applying phpstan changes, this should have been an !== since the begining. This fixes the relation between parents and childs on composer page.

Reply:
This one can also be covered with test with our current suite (but a panther test would do it bettrr

Reply:
This is now covered.